src/components/FlowRunsBarChart.vue

```diff
@@ -1,7 +1,8 @@
 <template>
-  <div ref="chart" class="flow-runs-bar-chart" :class="classes.root" :style="styles.root" @mouseleave="close">
```

Comment:
Removing this `@mouseleave="close"` fixes the issue where moving your mouse from the bar over to the popover closes the popover. 

however it introduces a UX regression in that the popover lingers onscreen until a click triggers `p-popover`'s `auto-close` behavior. 

Ideally the popover closes on mouseleave (of both the chart _and_ the popover contents) again. 
Needs some changes in prefect-design.